HotelBooking/HotelBooking.BusinessLogic/Dtos/GuestDto.cs

```diff
@@ -13,4 +11,6 @@ public class GuestDto
 
     public DateOnly DateBirth { get; set; }
 
+    public string Password { get; set; }
```

Comment:
dodałbym jeszcze pole dla ponownego wpisania hasła jak to często jest przy rejestracji, że trzeba wpisać 2 razy hasło (i muszą być takie same)

Reply:
Dodatkowo dorzuciłbym do fluentValidation (skoro go używamy) jakąs walidację na długość znaków hasła, żeby była jedna duża litera, cyfra i jakiś znak specjalny jak to zwykle jest dla haseł żeby było bezpieczne